docs/api/users-create-user.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+---
+
+# Create user
+
+Creates a [`user`](user) who is registering with the to-do service. The request body contains the new user details. You must specify the required properties for the user. 
+
+## URL
+
+```shell
+
+{POST}{server_url}/users/
+```
+
+## Request headers
+
+This request does not use any authorization. The endpoint is available to all users and applications.
+
+| Header name | Description | Required | Values |
+| -------------- | ------ | ------------ |------------ |
+| Content-Type | The format of the data to be posted. | Optional | application/json. Default value.  |
+| Accept | The format of the data to be returned. | Optional | application/json. Default value.|
+
+## Request body
+
+In the request body, specify a JSON representation of the [`user`](user) object. The following table lists the properties that are required when you create a user. 
+
+| Property | Description | Type | Required | Notes |
+| -------------- | ------ | ------------ |------------ |------------ |
+| last_name | The user's last name. | string | Required |   |
+| first_name | The user's first name. | string | Required |  |
+| email | The user's email. | string | Required | A unique email is required. |
+
+
+## Sample request
+
+The POST body should look something like this. You can change the values of each property as you’d like.
+
+```js
+[
+    {
+        "last_name": "Coughlan",
+        "first_name": "Sinead",
+        "email": "scough2@uw.edu",
+    }
+]
+```
+
+## Return body
+The following example shows the response. Note that the names should be the same as you used in your **Request body** and the response should include the new user’s id. The user’s id is automatically generated when the user is created.
+
+```js
+[
+    {
+        "id": "f0d3",
+        "last_name": "Coughlan",
+        "first_name": "Sinead",
+        "email": "scough2@uw.edu"
+    }
+]
+```
+
+## Return status
+
+| Status value | Return status | Description |
+| ------------- | ----------- | ----------- |
+| 201 | Created | User data created successfully. |
+| 500 | Internal server Error | Invalid JSON. |
+| ECONNREFUSED | N/A | Service is offline. Start the service and try again. |
+
+For more information about error handling, see  [Error handling codes](link to Mary Frances topic, tbd). 
+@Bob, another class member selected to write a topic about error handling. I intend to link users to it here when I know the filename etc.
```

Comment:
```suggestion
```
In an agile project, you'd leave this, and the previous line out of your PR so that the documentation is "complete" in that it doesn't have any TBA (to be added) content when it's published. i.e. it's as complete as it can be when it's merged.

After Mary Frances adds her Error Handling topic, she would let you know, or you would see her PR get merged, and then you could add these lines.

So, for now, leave them out, please and replace them with a single blank line.
The .MD files should end with an empty line. Not sure why, but I think it prevents or catches dangling markdown conversions.